### ISSUE TITLE
Ubiquity Installer not working

### ISSUE THREAD

**Issue by gity-up** [Distroshare/distroshare-ubuntu-imager#11]
The "Install" launcher within the Live Session does not work. Should it? If my OS ever became UN-bootable I would love to be able to reinstall it from my ISO file using the live session ubiquity GUI.

When I try running the command that the "Install" launcher would run from a terminal (sh -c 'ubiquity gtk_ui') I get this error:

The value for the SHELL variable was not found the /etc/shells file

This incident has been reported. (twice) :smile: because now YOU know.

Thanks in advance.

-ralphie

Update: Pretty sure this repo is unmaintained now. Been 2 months with no response. Script still works (mostly). Going to go experiment with a project called Systemback.

**Comment by eband** [comment]
I had a similar problem, except the Ubiquity installer icon does not even show!

**Comment by gity-up** [comment]
yeah sorry bud, i think this project is dead for now. if you haven't already, you should check out a distro called [manjaro](manjaro.org), they're cooking up some pretty neat stuff. [zorin](zorinos.com) is another tight project too. happy new year.

**Comment by eband** [comment]
Thanks for the reply, The smell or corpses were pickup up when I was at  
the door.

Are these two recommendations Ubuntu-based? At the moment, I'm comfortable  
with Ubuntu-based OSs.

Again, thanks.

**Comment by gity-up** [comment]
😄

zorin is. manjaro is not.

**Comment by cinnabarLinux** [comment]
This is still an issue in an otherwise incredible piece of software. Anyone know know a work around?? I have successfully created the .iso of my OS and any means of installing it would be just fine! Please let me know!
